addons/udes_get_info/tests/common.py

```diff
@@ -0,0 +1,136 @@
+"""Common code for testing get_info methods."""
```

Comment:
Some of these functions are defined in the common.py in udes_stock. I know this module does not depend on that. I have seen other modules also copying these functions again. Could we not start doing something better? Sorry to drag it on your story but it seems like a good place to start doing that.

For example, move the common functions in the common.py in udes_stock into udes_common which is a dependency of this module and then mix the classes where needed for the other modules that need those functions. It would simplify code all around the codebase and we would be using the same functions everywhere which is a lot cleaner and reliable long term. Even in terms of porting code.

Reply:
In general I agree.  I'm not sure it's going to happen in this PR though :).  Perhaps @PeterC10 or @adamrpatrick will chime in on this.

Reply:
Yep not enough scope in this story for this change.

Something we can look to handle once we get past our first milestone.